src/vendor/stubs/cx.js

```diff
@@ -24,7 +24,25 @@
  * @param [string ...]  Variable list of classNames in the string case.
  * @return string       Renderable space-separated CSS className.
  */
+
+'use strict';
+var warning = require('warning');
+
+var warned = false;
+
 function cx(classNames) {
+  if (__DEV__) {
+    if (!warned) {
+      warning(
+        warned,
+        'React.addons.classSet will be deprecated in a future version. See ' +
+        'the note linked in ' +
+        'http://facebook.github.io/react/docs/class-name-manipulation.html'
```

Comment:
Let's actually make this an `fb.me` link (`fb.me/react-addons-classset`?). That was we can update it whenever regardless of website changes. I think it'll fit on the previous line then too and we don't want more LOC :P